pandas/core/arrays/datetimes.py

```diff
@@ -1919,6 +1929,50 @@ def isocalendar(self) -> DataFrame:
         Index([1, 1], dtype='int32')
         """,
     )
+    qyear = _field_accessor(
+        "qyear",
+        "qy",
```

Comment:
```suggestion
        "qyear",
```
"qy" is not a valid field.